code/src/main/webapp/index.html

```diff
@@ -4,11 +4,25 @@
 <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <link rel="stylesheet" href="./CSS/style.css">
-   <title>First JSP</title>
+   <title>Welcome</title>
 </head>
 <body>
-   <h1>Well, life goes on ... </h1>
-   <p>(0.16450910536318764)</p>
-   <a href="./JSP/test.jsp"><h3>Try Again</h3></a>
+  <div id="wrapper">
+    <div id="title">
+      <h1> Welcome </h1>
+    </div>
+    <div id="intro">
+      <p> We are here to help with contact tracing!
+          If you have tested positive, we can help identify people
+          and places that you could have come in contact with.
+          We can also help if you are here to be notified of any possible contact
+          we someone who has tested positive
+      </p>
+    </div>
+    <div id="buttons">
```

Comment:
These stop working at smaller screen sizes. Please figure out how to make this look good at screen sizes as small as 320 PX.

Reply:
fixed